webrtc-stats.html

```diff
@@ -1866,8 +1875,7 @@ <h2>
               </dt>
               <dd>
                 <p>
-                  Only valid for video. It represents the total number of frames sent for this
-                  MediaStreamTrack.
+                  Represents the total number of frames sent for this MediaStreamTrack.
                 </p>
               </dd>
               <dt>
```

Comment:
The keyFramesSent definition still says "Only valid for video." This sentence can be removed.

Reply:
I'll fix these in a new patch in https://github.com/w3c/webrtc-stats/pull/273 (which has these patches), as rebasing is painful.